CharlieBackend.Api/Controllers/ImportController.cs

```diff
@@ -19,37 +19,43 @@ namespace CharlieBackend.Api.Controllers
     [ApiController]
     public class ImportController : ControllerBase
     {
-        private readonly IThemeXlsFileImporter _themeXlsFileImporter;
-        private readonly IStudentsGroupXlsxFileImporter _studentsGroupXlsxFileImporter;
+        IServiseImport _servise;
 
         /// <summary>
         /// Import controller constructor
         /// </summary>
-        public ImportController(IThemeXlsFileImporter themeXlsFileImporter,
-                                IStudentsGroupXlsxFileImporter studentsGroupXlsxFileImporter)
+        public ImportController(IServiseImport servise)
         {
-            _themeXlsFileImporter = themeXlsFileImporter;
-            _studentsGroupXlsxFileImporter = studentsGroupXlsxFileImporter;
+            _servise = servise;
         }
 
         /// <summary>
-        /// Imports group with students data from .xlsx file to database
+        /// Imports group with students data from .xlsx or csv file to database
         /// </summary>
         /// <response code="200">Successful import of data from file</response>
         /// <response code="HTTP: 400, API: 4">File validation error</response>
         [SwaggerResponse(200, type: typeof(IEnumerable<GroupWithStudentsDto>))]
         [Authorize(Roles = "Mentor, Secretary, Admin")]
-        [Route("group/{courseId}")]
+        [Route("groupCSV/{courseId}/{groupName?}/{startDate?}/{finishDate?}")]
```

Comment:
As for me, it looks ugly. Let's pass these params through query string instead of path.